@xen-orchestra/web-core/lib/components/console/VtsRemoteConsole.vue

```diff
@@ -80,6 +80,16 @@ async function createVncConnection() {
 
   vncClient.addEventListener('disconnect', handleDisconnectionEvent)
   vncClient.addEventListener('connect', handleConnectionEvent)
+  const canvas = consoleContainer.value?.querySelector('canvas') as HTMLCanvasElement | null
+  if (canvas) {
```

Comment:
Try to make the condition explicit. This improves readability
```suggestion
  if (canvas !== null) {
```